src/deploy-tes-on-azure/Deployer.cs

```diff
@@ -76,13 +69,18 @@ public class Deployer(Configuration configuration)
             .Handle<Azure.RequestFailedException>(azureException =>
                 (int)System.Net.HttpStatusCode.Conflict == azureException.Status &&
                 "OperationNotAllowed".Equals(azureException.ErrorCode))
-            .WaitAndRetryAsync(30, retryAttempt => System.TimeSpan.FromSeconds(10));
+            .WaitAndRetryAsync(30, retryAttempt => TimeSpan.FromSeconds(10));
 
         private static readonly AsyncRetryPolicy generalRetryPolicy = Policy
             .Handle<Exception>()
-            .WaitAndRetryAsync(3, retryAttempt => System.TimeSpan.FromSeconds(1));
+            .WaitAndRetryAsync(3, retryAttempt => TimeSpan.FromSeconds(1));
 
-        private static readonly System.TimeSpan longRetryWaitTime = System.TimeSpan.FromSeconds(15);
+        private static readonly TimeSpan longRetryWaitTime = TimeSpan.FromSeconds(15);
+
+        /// <summary>
+        /// Grants full access to manage all resources, but does not allow you to assign roles in Azure RBAC, manage assignments in Azure Blueprints, or share image galleries.
+        /// </summary>
+        private static readonly ResourceIdentifier All_Role_Contributor = AuthorizationRoleDefinitionResource.CreateResourceIdentifier(string.Empty, new("b24988ac-6180-42a0-ab88-20f7382dd24c"));
```

Comment:
Please include link to where this magic GUID came from

Reply:
Placing link here so that I can add it to the code: https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#general

Reply:
The ADAL-connected libraries provided a property that contained this guid with a nice name. The MSAL-connected libraries do not provide such a convenience.